store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/DoubleProxyTest.java

```diff
@@ -48,7 +48,7 @@ public class DoubleProxyTest {
 
     @BeforeEach
     public void setUpStores() throws OperationException {
-        SingleUseProxyMapStore.cleanUp();
+       FederatedStoreTestUtil.resetForFederatedTests();
```

Comment:
This should be caught automatically but indentation looks weird

Reply:
```suggestion
        FederatedStoreTestUtil.resetForFederatedTests();
```